tests/Integration/Behaviour/Features/Scenario/Manufacturer/manufacturer_management.feature

```diff
@@ -0,0 +1,78 @@
+      | name             | worst-shoes                                |
+      | short_description| Worst slippers in EU                       |
+      | meta_title       | Worst quality shoes                        |
+      | description      |                                            |
+      | meta_description | You'd better walk bare foot                |
+      | enabled          | false                                      |
+    Then manufacturer "shoeman" name should be "worst-shoes"
+    And manufacturer "shoeman" "short_description" in default language should be "Worst slippers in EU"
+    And manufacturer "shoeman" "description" field in default language should be empty
+    And manufacturer "shoeman" "meta_title" in default language should be "Worst quality shoes"
+    And manufacturer "shoeman" "meta_description" in default language should be "You'd better walk bare foot"
+    And manufacturer "shoeman" "meta_keywords" in default language should be "Boots, shoes, slippers"
+    And manufacturer "shoeman" should be disabled
+
+  Scenario: Enable and disable manufacturer status
+    Given manufacturer "shoeman" should be disabled
+    When I enable manufacturer "shoeman"
+    Then manufacturer "shoeman" should be enabled
+    When I disable manufacturer "shoeman"
+    Then manufacturer "shoeman" should be disabled
+
+  Scenario: Enabling multiple manufacturers in bulk action
+    When I add new manufacturer "baller" with following properties:
+      | name             | Baller                             |
+      | short_description| Makes big balls                    |
+      | description      | consectetur adipiscing elit. Morbi at nulla id mi gravida blandit a non erat. Mauris nec lorem vel odio sagittis ornare.|
+      | meta_title       | Have some balls                    |
+      | meta_description |                                    |
+      | meta_keywords    | Balls, basketball, football        |
+      | enabled          | false                              |
+    And I add new manufacturer "rocket" with following properties:
+      | name             | Rocket                             |
+      | short_description| Cigarettes manufacturer            |
+      | description      | Morbi at nulla id mi gravida blandit a non erat. Mauris nec lorem vel odio sagittis ornare.|
+      | meta_title       | You smoke - you die!               |
+      | meta_description | The sun is shining and the weather is sweet|
+      | meta_keywords    | Cigarettes, smoke                  |
+      | enabled          | true                               |
+    When I enable multiple manufacturers: "baller, rocket" using bulk action
+    Then manufacturers: "baller, rocket" should be enabled
+    When I disable multiple manufacturers: "baller, rocket" using bulk action
+    Then manufacturers: "baller, rocket" should be disabled
+
+  Scenario: Deleting manufacturer right after changing its name
```

Comment:
Is it some kind of special case when you delete "just after renaming it"?

Reply:
Yeah, it is just what matk suggested to do, to avoid a bit redudnat/short scenarios

Reply:
Deleting manufacturer alone is a valid scenario imo, but this works as well.

Reply:
https://github.com/PrestaShop/PrestaShop/pull/14132#discussion_r305856739